sdk/highlight-py/highlight_io/sdk.py

```diff
@@ -146,6 +147,7 @@ def my_fn():
 
         :param session_id: the highlight session that initiated this network request.
         :param request_id: the identifier of the current network request.
+        :param attributes: additional metadata to attribute to this error.
```

Comment:
Just curious, why call it `attributes` here and `metadata` in the JS SDKs?

Reply:
opentelemetry calls these tags `attributes`. ideally we should refactor the JS sdks to be consistent with their naming.